site/content/contribute/getting-started/branching.md

```diff
@@ -20,9 +20,12 @@ When your PR is required on a release branch, you will follow the cherry picking
 1. Add the appropriate milestone and the `CherryPick/Approved` label.
 1. When your PR is approved, it will be assigned back to you to perform the merge and any cherry picking if necessary.
 1. Merge the PR.
-1. Cherry pick the master commit back to the appropriate releases. If the release branches have not been cut yet, leave the labels as-is and cherry-pick once the branch has been cut. The release manager will remind you to finish your cherry-pick.
+1. An automated cherry pick process will try to cherry pick and then if success will create a new PR pointing to the correct release branch and set all the appropriate labels.
+1. If the automated cherry pick fails the developer will need to cherry pick manually. Cherry pick the master commit back to the appropriate releases. If the release branches have not been cut yet, leave the labels as-is and cherry-pick once the branch has been cut. The release manager will remind you to finish your cherry-pick.
 1. Set the `CherryPick/Done` label when completed.
 
+Note:
+  - If your PR need to go to more release branches you can type the following command in the PR comments: `/cherry-pick release-x.yz` and it will try to cherry pick to the branch you specified.
```

Comment:
```suggestion
  - If the PR need to go to other release branches, you can run the command `/cherry-pick release-x.yz` in the PR comments and it will try to cherry-pick it to the branch you specified.
```